src/test/java/org/valdroz/vscript/examples/ExampleCustomFunction.java

```diff
@@ -17,5 +16,27 @@ public static void main(String[] args) {
                 .eval(variantContainer);
 
         assertThat(result.asNumeric().doubleValue(), is(14.0));
+
+        masterRunBlock.registerFunction("if(condition, first, second)",
```

Comment:
`masterRunBlock.registerFunction("if(condition, first, second)"` should not be used as the function `if` is supposed to be handled natively by `vscript` just like  other inbuilt functions, e.g. `to_array`

Reply:
updated. We can discuss tomorrow if the changes are still incorrect. Please let me know. Thanks